### PR TITLE
Make urls use bps sans ttfb

### PR DESCRIPTION
**Pull request by garrettmoon**

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/1e1033495718d10f12048920c976285c/raw/f3954bf73c8bb260c0c5576d8485102bcd815e5e/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/f8a3e84d5c3a3e9b8cb261f9db594c8d/raw/187b82f6a269b3252273419a1be21971bac278e5/log.txt)

File: Source/Classes/PINProgressiveImage.h [hunk 1]
```diff
@@ -33,21 +33,30 @@
...
- @return NSData the current data for the image
+ @return NSData The current data for the image.
  */
 - (nullable NSData *)data;
 
+/**
+ Returns bytes per second adjusted by subtracting subtraction from the task length before calculation.
+ Useful when you want something like bytesPerSecond discounting the time to first byte.
+ 
+ @param subtraction an amount to be subtracted from task length befor calculation.
```
Comment (Adlai-Holler) [inline Source/Classes/PINProgressiveImage.h]: Capital `An`, `before calculation`.

File: Source/Classes/PINProgressiveImage.m [hunk 2]
```diff
@@ -128,6 +128,31 @@ - (float)l_bytesPerSecond
     return downloadedBytes / taskLength;
 }
 
+- (float)adjustedBytesPerSecond:(CFTimeInterval)subtraction
```
Comment (Adlai-Holler) [inline Source/Classes/PINProgressiveImage.m]: `bytesPerSecondWithTimeToFirstByte:`?

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/ea7255a498bceaddb07f951038e9982e/raw/43774d00487f3af20f897bc74da5ea9b45ee2a7a/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/6f14d4856c8fe5f2b5d17d15e813cd1b/raw/a4f42f81d1ca69fd7b4ca46ed442c26a5e724913/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/f71722b667a930d29d6785ae82ab6136/raw/17c7926bd0eda14799d6dedf648571fbc9b37124/log.txt)

**Comment by pinterest-ios** [comment]
🚫 CI failed with [log](https://gist.githubusercontent.com/pinterest-ios/7b1a09679356556bfaf5222273f064b3/raw/0bf11e8c8990b61838b83c947289ce8d173f8948/log.txt)

**Pull request merged**